schemas/org.gnome.shell.extensions.mpris-label.gschema.xml

```diff
@@ -30,6 +30,14 @@
       <default>"＿"</default>
     </key>
 
+    <key name="font-color" type="s">
+      <default>""</default>
+    </key>
+
+    <key name="font-color-default" type="s">
+      <default>""</default>
+    </key>
+
```

Comment:
Unused, leftover from [e6ddde9](https://github.com/Moon-0xff/gnome-mpris-label/pull/94/commits/e6ddde929cbbeec92a8f32c342dee58ffb8f8f3e) and merged to `main`.